src/hostapi/pulseaudio/pa_linux_pulseaudio_cb.c

```diff
@@ -827,16 +832,19 @@ PaError PaPulseAudio_StartStreamCb( PaStream * s )
 
     if( stream->outputStream )
     {
-        /* tlength does almost the same as fragsize in record.
-         * See reasoning up there in comments.
+        /* The tlength parameter functions similarly to fragsize in
+         * the recording process, as explained in the preceding comments.
          *
-         * In future this should we tuned when things changed
-         * this just 'good' default
+         * In the future, this should be adjusted as necessary when
```

Comment:
What does "in the future" mean? is this a `TODO` or `FIXME`? what does "conditions change" mean? what conditions?